src/main/java/de/cas/mse/exercise/csvdb/impl/FileStorage.java

```diff
@@ -0,0 +1,47 @@
+package de.cas.mse.exercise.csvdb.impl;
+
+import java.util.List;
+
+import de.cas.mse.exercise.csvdb.Storage;
+
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileStorage implements Storage {
+
+    protected final Path path;
+
+    public FileStorage(Path path) {
+        this.path = path;
+    }
+
+    public List<String> read() {
+        try {
+            return Files.readAllLines(path);
+        } catch (final IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public void write(List<String> input) {
```

Comment:
insert or append would fit better

Reply:
Valid. Depending on how abstract and reusable this "FileStorage" is or if we really want it to be a CsvFileStorage.